dspfparser/src/main/kotlin/com/smeup/dspfparser/linesprocessor/LinesSubstringsMerger.kt

```diff
@@ -14,24 +14,24 @@ internal fun merge(linesSubstrings: MutableList<DSPFLineSubstrings>): DSPFLineSu
     linesSubstrings.forEach {
         if (it.continuationChar != null) {
             if (continuesAtAnyColumn == null) {
-                count = it.count!!
-                line += it.line!!.removeLast(it.continuationChar!!.toString())
+                count = it.count
+                line += it.line.removeLast(it.continuationChar.toString())
```

Comment:
Concat using `line +=` could be inefficient. Replace it with `StringBuilder.append` or equivalent.